### DISCUSSION TITLE
Fast way to go to the next and previous Post

### DISCUSSION THREAD

**Discussion by gerrited** [hugo-sid/hugo-blog-awesome#156] [Feature requests]
Hi!

I am playing around with the theme and think it would be great, If there were the option at the end of a post to go to the next or previous post. This would make navigation easier and users could see more interesting content.

What do you think about it?

Gerrit

**Comment by hugo-sid** [comment]
> If there were the option at the end of a post to go to the next or previous post. This would make navigation easier and users could see further contents.

Hi @gerrited, this is a nice idea indeed. I will try to implement this in next few days.

**Comment by matrixik** [comment]
Hi @hugo-sid you can check how Minimo theme is generating prev/next post link (with post titles so reader know what he is clicking): https://github.com/MunifTanjim/minimo/blob/master/layouts/partials/nav/entry_nav.html#L16-L40
